docs/conflicts.md

```
@@ -1,72 +1,120 @@
-Note that on a conflicting transaction, the delta-apply column will be correctly calculated and applied.
+Note that on a conflicting transaction, the delta-apply column will be
+correctly calculated and applied.
 
-To make a column a conflict-free delta-apply column, ensuring that the value replicated is the delta of the committed changes (the old value plus or minus any new value) to a given record, you need to apply the following settings to the column:  `log_old_value=true, delta_apply_function=spock.delta_apply`.  For example:
+To make a column a conflict-free delta-apply column, ensuring that the value
+replicated is the delta of the committed changes (the old value plus or
+minus any new value) to a given record, you need to apply the following
+settings to the column: `log_old_value=true,
+delta_apply_function=spock.delta_apply`. For example:
 
 ```sql
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)